### DISCUSSION TITLE
Discord Webhook

### DISCUSSION THREAD

**Discussion by maurxce** [outline/outline#6133] [Self Hosting] [answered]
Hello! Are there any resources on how to set up a discord webhook?
I tried it myself by doing this:
`Preferences > Webhooks > New Webhook`
  - Name: Discord
  - URL: <URL> (tried both normal and with /slack)
  - All events
But when I published a note I didn't get any message

**Comment by tommoor** [comment, marked as answer]
It is not possible to just send Outline webhooks to Discord as the shape of the request is different. For this, someone would need to build a Discord integration based on the Slack integration

**Reply by maurxce** [reply]
Yea I noticed that too after I compared the request body of the webhook to a slack webhook request body.

I was originally contemplating whether I should write a "converter" if there's no native support, but considering the amount of events and the little added value it has, I will probably not be doing that.

I appreciate the answer and will close this discussion now!^^

**Comment by maximjsx** [comment]
https://github.com/NextFightNetwork/OutlineToDiscord
Had the same issue so...
If anyone is still interested
